anomali-limo-feed/info.json

```diff
@@ -0,0 +1,295 @@
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "operation": "get_objects_by_collection_id",
+      "title": "Get Objects By Collection ID",
+      "description": "Retrieve all available objects in a Collection.",
+      "category": "investigation",
+      "annotation": "get_objects_by_collection_id",
+      "api_output_schema": "get_output_schema",
+      "output_schema": {},
+      "parameters": [
+        {
+          "title": "Collection ID",
+          "name": "collectionID",
+          "visible": true,
+          "required": true,
+          "editable": true,
+          "type": "text",
+          "tooltip": "Returns general information about the collection, such as id, title, description, can_read, can_write and media_types.",
+          "description": "Returns general information about the collection, such as id, title, description, can_read, can_write and media_types."
+        },
+        {
+          "title": "Created After",
+          "name": "added_after",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "datetime",
+          "tooltip": "Filter the result set to only include the items added after the specified time.",
+          "description": "Filter the result set to only include the items added after the specified time."
+        },
+        {
+          "title": "Save Response to File",
+          "name": "file_response",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "checkbox",
+          "value": true,
+          "onchange": {
+            "true": [
+              {
+                "title": "Filename",
+                "name": "filename",
+                "type": "text",
+                "visible": true,
+                "editable": true,
+                "required": true,
+                "placeholder": "Eg. demo.json",
+                "tooltip": "The file will be created under /tmp"
+              }
+            ]
+          },
+          "tooltip": "If enabled, the feed will be written to a value, else returned as a JSON",
+          "description": "Specify the response field, If check then save the response in File else in JSON"
+        },
+        {
+          "title": "Offset",
+          "name": "offset",
+          "type": "text",
+          "visible": true,
+          "editable": true,
+          "required": true,
+          "placeholder": "0",
+          "value": "0",
+          "tooltip": "From which offset you want to fetch indicators",
+          "description": "From which offset you want to fetch indicators"
+        },
+        {
+          "title": "Limit",
+          "name": "limit",
+          "type": "text",
+          "visible": true,
+          "editable": true,
+          "required": false,
+          "placeholder": "1000",
+          "tooltip": "Upper limit of indicators to fetch, if not provided then fetch the full objects in specified collection.",
```

Comment:
Maximum number of indicators to fetch. If not specified, all objects in the specified collection are fetched.